diffpy/pdfmorph/pdfmorphapp.py

```diff
@@ -88,53 +76,82 @@ def custom_error(self, msg):
         action="store_true",
         help=f"""Changes usage to \'{prog_short} [options] FILE DIRECTORY\'. FILE
 will be morphed with each file in DIRECTORY as target.
 Files in DIRECTORY are sorted by alphabetical order unless a field is
-specified by --sort-by. See -s and Plot Options for how saving and
-plotting change when this option is enabled.""",
+specified by --sort-by.""",
     )
-    parser.add_option(
+    group.add_option(
         '--sort-by',
         metavar="FIELD",
         dest="field",
         help="""Used with --multiple to sort files in DIRECTORY by FIELD from lowest to highest.
 FIELD must be included in the header of all the PDF files.""",
     )
-    parser.add_option(
+    group.add_option(
         '--reverse',
         dest="reverse",
         action="store_true",
         help="""Sort from highest to lowest instead.""",
     )
-    parser.add_option(
+    group.add_option(
         '--serial-file',
-        metavar="SERIAL",
+        metavar="SERIALFILE",
         dest="serfile",
         help="""Look for FIELD in a serial file instead.
-Must specify name of serial file SERIAL.""",
-    )
-    parser.add_option(
-        '--rmin', type="float", help="Minimum r-value to use for PDF comparisons."
+Must specify name of serial file SERIALFILE.""",
     )
-    parser.add_option(
-        '--rmax', type="float", help="Maximum r-value to use for PDF comparisons."
-    )
-    parser.add_option(
-        '--pearson',
-        action="store_true",
-        dest="pearson",
-        help="Maximize agreement in the Pearson function. Note that this is insensitive to scale.",
+    group.add_option(
+        '--save-names-file',
+        metavar="NAMESFILE",
+        dest="snamesfile",
+        help=f"""Used when both -s and --multiple are enabled.
+Specify names for each manipulated PDF when saving (see -s) using a serial file
+NAMESFILE. The format of NAMESFILE should be as follows: each target PDF
+is an entry in NAMESFILE. For each entry, there should be a key {__save_morph_as__}
+whose value specifies the name to save the manipulated PDF as.
+(See sample names files in the PDFmorph tutorial).""",
     )
-    parser.add_option(
-        '--addpearson',
-        action="store_true",
-        dest="addpearson",
-        help="""Maximize agreement in the Pearson function as well as
-minimizing the residual.""",
+    group.add_option(
+        '--plot-parameter',
+        metavar='PLOTPARAM',
+        dest='plotparam',
+        help="""Used when both plotting and --multiple are enabled.
+Choose a PLOTPARAM to plot for each morph (i.e. adding --pp=Pearson means the program
+will display a plot of the Pearson correlation coefficient for each morph-target pair).
+PLOTPARAM is not case sensitive, so both Pearson and pearson indicate the same parameter.
+When PLOTPARAM is not specified, Rw values for each morph-target pair will be plotted.
+PLOTPARAM will be displayed as the vertical axis label for the plot.""",
     )
```

Comment:
Only one new option added this commit. All others were just moved (put into separate options group)